NEWS.md

```diff
@@ -45,7 +45,8 @@
 </details>
 
 # admiral 1.1.1
-
+- `derive_vars_atc()` and `create_single_dose_dataset()` `by_vars` argument updated to use `get_admiral_option("subject_keys")` instead of  `USUBJID` or `STUDYID`.
```

Comment:
These updates should go under the development version section of the news.  When we release it will change to admiral 1.1.2

Reply:
Done! Pushing now.